validation/validationRun/validationRun.py

```diff
@@ -234,10 +234,12 @@
     comparisonFilenames.append('val.comparison' + comparePostfix)
 
 if doReferenceRun:
+    call(['mkdir', 'reference/'])
```

Comment:
make this a `makedir -p` to avoid any warnings if directory already exists

Reply:
will do